infra-templates/prom/config.yml

```diff
@@ -1,7 +1,7 @@
 name: Prometheus Agents
 description: |
   Stack to run Prometheus agents to monitor a Rancher environment
-version: '0.1'
+version: '0.2'
```

Comment:
We usually keep the same version to avoid having to duplicate the YAML files.

Reply:
Hmmm.... there's something I don't understand about this. What do you mean avoid duplicating the YAML files? Which ones? Isn't it necessary anyway for each new "version" you create with a successive numbered folders?

Reply:
I think it's a bit of a hack, but you can simply update the `docker-compose.yml` and  `rancher-compose.yml` of a stack in-place, keeping the same version number, and still use it from Rancher. See c0ccfb3dbd0bc738ea370d1ebb4f9bb6dc1995aa for an example.

Rancher's UI won't show the "Update available" label for the stacks, but you can hit the `Up to date` label next to the stack and that'll trigger an upgrade to the new definition anyways.

Reply:
Ah, I see. Yes, I did find that commit a bit weird. It's good to know you can force update if necessary.